store/tikv/mock-tikv/cluster_manipulate.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mocktikv
+
+import "fmt"
+
+// BootstrapWithSingleStore initializes a Cluster with 1 Region and 1 Store.
+func BootstrapWithSingleStore(cluster *Cluster) {
```

Comment:
@coocood A lot more functions (LeaderChange, KillNode, etc.) will be added later. They are considered as more higher level batch operations applied on the Cluster.